### PR TITLE
Elo equality

### PR DESCRIPTION
**Pull request by Isylwin**

### CODE REVIEW

File: common/src/main/nl/jft/common/util/elo/EloExpectation.java [hunk 1]
```diff
@@ -108,4 +108,34 @@ public double getSecondDelta() {
...
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (!(obj instanceof EloExpectation)) {
+            return false;
+        } else {
+            EloExpectation other = (EloExpectation) obj;
+
+            return firstElo == other.firstElo &&
```
Comment (TehMessiah) [inline common/src/main/nl/jft/common/util/elo/EloExpectation.java]: CRITICAL Reduce the number of conditional operators (6) used in the expression (maximum allowed 3). [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1067)
MAJOR Equality tests should not be made with floating point values. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1244)

File: common/src/main/nl/jft/common/util/elo/EloExpectation.java [hunk 2]
```diff
@@ -108,4 +108,34 @@ public double getSecondDelta() {
...
+                    secondElo == other.secondElo &&
```
Comment (TehMessiah) [inline common/src/main/nl/jft/common/util/elo/EloExpectation.java]: MAJOR Equality tests should not be made with floating point values. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1244)

File: common/src/main/nl/jft/common/util/elo/EloExpectation.java [hunk 3]
```diff
@@ -108,4 +108,34 @@ public double getSecondDelta() {
...
+                    firstGoalsScored == other.firstGoalsScored &&
+                    secondGoalsScored == other.secondGoalsScored &&
+                    maxGoals == other.maxGoals &&
+                    firstDelta == other.firstDelta &&
```
Comment (TehMessiah) [inline common/src/main/nl/jft/common/util/elo/EloExpectation.java]: MAJOR Equality tests should not be made with floating point values. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1244)

File: common/src/main/nl/jft/common/util/elo/EloExpectation.java [hunk 4]
```diff
@@ -108,4 +108,34 @@ public double getSecondDelta() {
...
+                    secondDelta == other.secondDelta;
```
Comment (TehMessiah) [inline common/src/main/nl/jft/common/util/elo/EloExpectation.java]: MAJOR Equality tests should not be made with floating point values. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1244)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 5 issues
* CRITICAL 1 critical
* MAJOR 4 major

Watch the comments in this conversation to review them.

**Comment by Isylwin** [comment]
I wiel fix at home. I iz great fieeexer

File: common/src/main/nl/jft/common/util/Numbers.java [hunk 5]
```diff
@@ -0,0 +1,37 @@
+package nl.jft.common.util;
+
+/**
+ * {@code Numbers} is an utility class for primitive number types.
+ *
+ * @author Oscar de Leeuw
+ */
+public class Numbers {
```
Comment (TehMessiah) [inline common/src/main/nl/jft/common/util/Numbers.java]: MAJOR Add a private constructor to hide the implicit public one. [rule](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1118)

**Comment by TehMessiah** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**